src/solvers/strings/string_format_builtin_function.cpp

```diff
@@ -17,10 +17,10 @@ Date:   June 2019
 
 #include "format_specifier.h"
 #include "string_format_builtin_function.h"
+#include "string_refinement_util.h"
```

Comment:
@romainbrenguier Do you see any problem with this added dependency?

Reply:
I think it's ok, but we ideally `string_refinement_util.h` should be cleaned-up to only have utility functions, and the data types inside should be moved to other files.

Reply:
That's the plan. Hopefully we can free some time to do that.